partials/benefit-estimate.html

```diff
@@ -1,226 +1,228 @@
 <div id=benefitEstimate class="benefit-estimate fixed-center-column"
      ng-show="showReport()" ng-cloak>
 
-  <h2 id=nav-nra>Normal Retirement Age</h2>
-  <p>
-    Your primary insurance amount rounded down to the next lower whole dollar
-    is <b>${{recipient.primaryInsuranceAmountFloored() | number:0}}</b> / month
-    and is the benefit you will earn if you retire at your
-    <u>normal retirement age</u> (NRA). Normal Retirement Age is between 65
-    and 67, depending on when you were born. You may also see this
-    referred to as the <u>full retirement age</u> (FRA).
-  </p>
+  <div ng-show="recipient.isEligible()">
```

Comment:
Rather than inserting a new div here, I'd just add the constraint to the parent div. Something like:

```
<div id=benefitEstimate class="benefit-estimate fixed-center-column"
      ng-show="showReport() && recipient.isEligible()" ng-cloak>
```

I suppose both work, but it reduces the nesting.

This file has a big diff, I think because of the added whitespace? I don't think you made any other changes in this file other than this added nested diff, correct?